lib/Github/Api/Search.php

```diff
@@ -40,9 +40,9 @@ public function repositories($q, $sort = 'updated', $order = 'desc')
      *
      * @return array list of issues found
      */
-    public function issues($q, $sort = 'updated', $order = 'desc')
+    public function issues($q, $sort = 'updated', $order = 'desc', $page = 1)
```

Comment:
The page parameter should not be passed here. You need to use the result pager class.